### ISSUE TITLE
background image not appearing

### ISSUE THREAD

**Issue by xsilent-cleopatra** [aFarkas/lazysizes#732]
**Describe the bug**
Actualy not bug, maybe I missed something. 

**Steps to reproduce the behavior:**
Provide all necessary steps to reproduce the bug.
What I have tried is at [jsfiddle here](https://jsfiddle.net/aalfiann/t58osbrw/5/).

**What is the expected behavior**
I want to be responsive so I don't want to set height or width, just use actual image size.

**What happened instead**
Image not appearing.

**In what environment (browser/device etc.) does this bug happen/not happen:**
Latest Chrome browser

**Comment by xcartmods** [comment]
add width="650" height="800" to the image.

**Comment by xsilent-cleopatra** [comment]
yes it is work, but is there features to make this auto resize on mobile and desktop?
